Yank/tests/test_restraints.py

```diff
@@ -290,7 +290,8 @@ def test_partial_parametrization():
                 particles, _ = force.getBondParameters(0)
                 assert particles == tuple(restraint.restrained_receptor_atoms + restraint.restrained_ligand_atoms)
             # RMSD restraint.
-            elif OpenMM73.dev_validate and isinstance(force, openmm.CustomCVForce):
+            elif OpenMM73.dev_validate and isinstance(force, openmm.CustomCVForce) and \
```

Comment:
If remember correctly, the `OpenMM73.dev_validate` we added some time ago is unnecessary now since the CV force is openmm>=7.2 and we've pinned it in the conda recipe, correct?

Reply:
RMSD Force is still a 7.3 thing, but I need to fix that to be more generic

Reply:
Ah! I see, I forgot about the RMSD restraint.